docs/source/notebook.rst

```diff
@@ -95,20 +95,23 @@ and open a web browser to the URL of the web application (by default,
 ``http://127.0.0.1:8888``).
 
 The landing page of the Jupyter notebook web application, the **dashboard**,
-shows the notebooks currently available in the notebook directory (by default,
-the directory from which the notebook server was started).
+shows the files and folders (subdirectories) in the notebook directory,  which by default is
+the directory from which the notebook server was started.
```

Comment:
To emphasize notebooks and dashboard layout, I suggest:

...shows the content of the notebook directory which may include notebooks, files, and folders. The notebook directory defaults to the directory where ``jupyter notebook`` was started or a location, if specified, in the Jupyter configuration file.